src/displayapp/screens/InfiniPaint.cpp

```diff
@@ -7,9 +7,9 @@ using namespace Pinetime::Applications::Screens;
 extern lv_font_t jetbrains_mono_extrabold_compressed;
 extern lv_font_t jetbrains_mono_bold_20;
 
-InfiniPaint::InfiniPaint(Pinetime::Applications::DisplayApp *app, Pinetime::Components::LittleVgl& lvgl) : Screen(app), lvgl{lvgl} {
```

Comment:
Not a big deal, but I prefer `Pinetime::Components::LittleVgl& lvgl` than `Pinetime::Components::LittleVgl &lvgl`.

Reply:
Do you prefer ` *` or `* `  or ` * `?

Reply:
I updated the rules to fix it.